gitea/config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "arch": [
+    "aarch64",
+    "amd64"
+  ],
+  "codenotary": "alexandrep.github@gmail.com",
+  "description": "Gitea for Home Assistant",
+  "image": "ghcr.io/alexbelgium/gitea-{arch}",
+  "init": false,
+  "map": [
+    "share:rw",
+    "ssl:ro"
+  ],
+  "name": "Gitea",
+  "options": {
+    "certfile": "fullchain.pem",
+    "keyfile": "privkey.pem",
+    "ssl": false
+  },
+  "ports": {
+    "3000/tcp": 3000,
+    "2222/tcp": 2222
+  },
+  "ports_description": {
+    "3000/tcp": "Http web interface",
+    "22/tcp": "Ssh"
```

Comment:
Hi, there is a mismatch : in port_description your have 22 but in ports you have 2222? thanks!

Reply:
I changed it to 22 => 2222